### PR TITLE
[CHAD-3909] SmartSense Multi Sensor: Add preference translations

### PR DESCRIPTION
**Pull request by greens**
… smartsense garage multi

### CODE REVIEW

File: devicetypes/smartthings/smartsense-garage-door-multi.src/smartsense-garage-door-multi.groovy [hunk 1]
```diff
@@ -71,7 +71,7 @@ metadata {
 	}
 
 	preferences {
-		input "tempOffset", "number", title: "Temperature offset", description: "Select how many degrees to adjust the temperature.", range: "*..*", displayDuringSetup: false
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-garage-door-multi.src/smartsense-garage-door-multi.groovy]: `Select how many degrees to adjust the temperature.` is the correct `description`

File: devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy [hunk 2]
```diff
@@ -67,7 +67,7 @@ metadata {
 			input "tempOffset", "number", title: "Temperature Offset", description: "Adjust temperature by this many degrees", range: "*..*", displayDuringSetup: false
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy]: `Offset` should be changed to `offset`

**Comment by tpmanley** [comment]
`SmartSense Garage Door Multi` needs updates as well I believe

**Comment by greens** [comment]
@tpmanley I'm pretty sure it's correct as-is?

https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/smartsense-garage-door-multi.src/i18n/messages.properties
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/smartsense-garage-door-multi.src/smartsense-garage-door-multi.groovy

**Comment by tpmanley** [comment]
@greens , you're right, my mistake

**Review by tpmanley — APPROVED** [review]

**Pull request merged**